src/NHibernate.Test/NHSpecificTest/NH1274ExportExclude/NH1274ExportExcludeFixture.cs

```diff
@@ -7,6 +7,7 @@
 using NHibernate.Criterion;
 using NHibernate.Engine;
 using NHibernate.Tool.hbm2ddl;
+using NHibernate.Util;
```

Comment:
We wish to remove NHibernate collection custom extensions, so please do not add more usages of them.